Source/WebKit/WebProcess/Extensions/API/WebExtensionAPIBookmarks.h

```diff
@@ -35,6 +35,30 @@ namespace WebKit {
 class WebExtensionAPIBookmarks : public WebExtensionAPIObject, public JSWebExtensionWrappable {
     WEB_EXTENSION_DECLARE_JS_WRAPPER_CLASS(WebExtensionAPIBookmarks, bookmarks, bookmarks);
 
+public:
+#if PLATFORM(COCOA)
+    void createBookmarks(NSDictionary *bookmark, Ref<WebExtensionCallbackHandler>&&, NSString **outExceptionString);
+
+    void getChildren(NSString *id, Ref<WebExtensionCallbackHandler>&&, NSString **outExceptionString);
+
+    void getRecent(long long numberOfItems, Ref<WebExtensionCallbackHandler>&&, NSString **outExceptionString);
+
+    void getSubTree(NSString *id, Ref<WebExtensionCallbackHandler>&&, NSString **outExceptionString);
+
+    void getTree(Ref<WebExtensionCallbackHandler>&&, NSString **outExceptionString);
+
+    void get(NSObject *idOrIdList, Ref<WebExtensionCallbackHandler>&&, NSString **outExceptionString);
+
+    void move(NSString *id, NSDictionary *destination, Ref<WebExtensionCallbackHandler>&&, NSString **outExceptionString);
+
+    void remove(NSString *id, Ref<WebExtensionCallbackHandler>&&, NSString **outExceptionString);
+
+    void removeTree(NSString *id, Ref<WebExtensionCallbackHandler>&&, NSString **outExceptionString);
+
+    void search(NSObject *query, Ref<WebExtensionCallbackHandler>&&, NSString **outExceptionString);
+
+    void update(NSString *id, NSDictionary *changes, Ref<WebExtensionCallbackHandler>&&, NSString **outExceptionString);
```

Comment:
Super-nit: I think instead of doing a blank line between each of these, they could be grouped by type of method.

`create`

`get*`

`remove*`

`search`
`update`
`move`.